pkg/sdk/session_policies_def.go

```diff
@@ -0,0 +1,110 @@
+package sdk
+
+import g "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/poc/generator"
+
+//go:generate go run ./poc/main.go
+
+var SessionPoliciesDef = g.NewInterface(
+	"SessionPolicies",
+	"SessionPolicy",
+	g.KindOfT[SchemaObjectIdentifier](),
+).
+	CreateOperation(
+		"https://docs.snowflake.com/en/sql-reference/sql/create-session-policy",
+		g.QueryStruct("CreateSessionPolicy").
+			Create().
+			OrReplace().
+			SQL("SESSION POLICY").
+			IfNotExists().
+			Name().
+			OptionalIntAssignment("SESSION_IDLE_TIMEOUT_MINS", g.ParameterOptions().NoQuotes()).
```

Comment:
Since we decided to have "Text" instead of "String" I would also consider "Number" instead of "Int", but that's nitpicking and either works for me

Reply:
I will change it, you are right

Reply:
Done.